src/components/member-profile/index.js

```diff
@@ -358,7 +358,7 @@ const Profile = (props) => {
           <div className={classNames.memberDetails}>
             <motion.img
               layoutId={username}
-              src={imageLink.w_200}
+              src={imageLink.w_200 ?? '/icons/user.png'}
```

Comment:
Not the ideal way to handle the error. You can check how we have handled the error in other files. [For reference](https://github.com/Real-Dev-Squad/website-members/blob/45dc8cae73e7b03152030e15230d0ecad0592bb6/src/components/member-card/index.js?_pjax=%23js-repo-pjax-container%2C%20div%5Bitemtype%3D%22http%3A%2F%2Fschema.org%2FSoftwareSourceCode%22%5D%20main%2C%20%5Bdata-pjax-container%5D#L35)

Reply:
Sure.